conda-recipes/llvmlite/build.sh

```diff
@@ -19,11 +24,21 @@ if [[ $(uname) == Darwin ]]; then
 fi
 
 if [ -n "$MACOSX_DEPLOYMENT_TARGET" ]; then
-    # OSX needs 10.7 or above with libc++ enabled
-    export MACOSX_DEPLOYMENT_TARGET=10.10
+    if [[ $build_platform == osx-arm64 ]]; then
+	export MACOSX_DEPLOYMENT_TARGET=11.0
+    else
+	# OSX needs 10.7 or above with libc++ enabled
+	export MACOSX_DEPLOYMENT_TARGET=10.10
+    fi
```

Comment:
```suggestion
    if [[ $build_platform == osx-arm64 ]]; then
        export MACOSX_DEPLOYMENT_TARGET=11.0
    else
        # OSX needs 10.7 or above with libc++ enabled
        export MACOSX_DEPLOYMENT_TARGET=10.10
    fi
```
swap tabs for spaces.